controllers/admin/AdminImportController.php

```diff
@@ -72,7 +72,7 @@ class AdminImportControllerCore extends AdminController
 		'available_now' => array('AdminImportController', 'createMultiLangField'),
 		'available_later' => array('AdminImportController', 'createMultiLangField'),
 		'category' => array('AdminImportController', 'split'),
-		'online_only' => array('AdminImportController', 'getBoolean')
+		'online_only' => array('AdminImportController', 'getBoolean'),
```

Comment:
Hi,

In php yes you can do it normally. Some think it helps the next commiter to add rows. Like holding the door in the sub.

Regards